docs/recipes/vue.md

```diff
@@ -60,6 +80,71 @@ test('renders', t => {
 });
 ```
 
+## Sample data test
+
+```js
+import test from 'ava';
+import Vue from 'vue';
+import Component from 'component.vue';
+
+test('it has a message', t => {
+	const vm = new Vue(Component).$mount();
+	t.is('my message', vm.message);
+});
+```
+
+## Sample DOM testing with data manipulation
+
+When manipulating the data object of a Vue model, the DOM will not be updated immediately. It will be changed on the next tick.
+
+```js
+import test from 'ava';
+import Vue from 'vue';
+import Component from 'component.vue';
+
+test('see the updated message', t => {
+	const vm = new Vue(Component).$mount();
+	t.is(vm.$el.textContent, 'my message');
+
+	vm.message = 'my new message';
+	// this fails here: t.is('my new message', vm.$el.textContent)
+
+	Vue.nextTick(() => {
```

Comment:
I assume this is asynchronous? That'll mean the assertion is run when the test is ended. We don't currently warn about that, but it means the assertion is ignored. You'd have to get a promise out of this or use `test.cb()` with `t.end()`.

Reply:
Agree with using async and await syntax.